docs/getting-started/prerequisite.md

```diff
@@ -212,30 +213,28 @@ the kernel SCSI subsystem translates SCSI commands into ATA commands.
 | Utility Name     | SCSI Command Invoked   | Description                       |
 | :--------------- | :--------------------- | :-------------------------------- |
 | **sg_rep_zones** | REPORT ZONES           | Get a ZBC disk's zone information |
-| **sg_reset_wp**  | RESET WRITE POINTER    | Reset one zone or all zones on a ZBC disk |
-| **sg_zone**      | CLOSE ZONE, FINISH ZONE, OPEN ZONE | Sends one of these commands to the specified ZBC disk |
+| **sg_reset_wp**  | RESET WRITE POINTER    | Reset one zone or all zones of ZBC disk |
+| **sg_zone**      | CLOSE ZONE, FINISH ZONE, OPEN ZONE | Sends one of these commands to the given ZBC disk |
 
 </center>
 
-See the sg3_utils section under [SCSI Generic
-Utilites](../projects/sg3utils.md#sg3_utils) for sg3_utils usage examples. 
-
+[This section](../projects/sg3utils.md#sg3_utils) shows some examples of these
+utilities execution
```

Comment:
Missing period at the end of the sentence. I will fix this when applying. No need to resend.